packages/nexrender-server/src/routes/jobs-pickup.js

```diff
@@ -39,6 +39,23 @@ module.exports = async (req, res) => {
                 if (isNaN(b.priority)) b.priority = 0
                 return b.priority - a.priority
             })[0]
+        } else if (process.env.NEXRENDER_ORDERING === 'stage-distributed') {
+            const { jobs } = queued.reduce((res, item) => {
+                const stage = item.ct?.attributes?.stage;
+                if (!stage) {
+                    res.jobs.push(item)
```

Comment:
Please change to select first (oldest) here - this will be needed for `default` job types (renders).

Reply:
updated